snuba/datasets/errors_replacer.py

```diff
@@ -55,6 +55,18 @@ class ExcludeGroups:
 QueryTimeFlags = Union[NeedsFinal, ExcludeGroups]
 
 
+@dataclass
```

Comment:
We normally make these frozen unless it needs to be mutable (it doesn't seem like that's the case right now)